bip-0373.mediawiki

```diff
@@ -173,16 +172,16 @@ Signers must remember to apply any relevant tweaks such as a tweak that is the r
 BIP 32 unhardened derivation with the aggregate public key as the parent key.
 
 If all other signers have provided a <tt>PSBT_IN_MUSIG2_PARTIAL_SIG</tt>, then the final signer may
-perform the <tt>PartialSigAgg</tt> algorithm and produce a BIP 340 compatible signature that can be
-placed into a <tt>PSBT_IN_TAP_KEY_SIG</tt> or a <tt>PSBT_IN_TAP_SCRIPT_SIG</tt>.
+perform the <tt>PartialSigAgg</tt> algorithm and produce a [[bip-0340.mediawiki|BIP 340]] compatible 
+signature that can be placed into a <tt>PSBT_IN_TAP_KEY_SIG</tt> or a <tt>PSBT_IN_TAP_SCRIPT_SIG</tt>.
 
 ===Finalizer===
 
 A finalizer may perform the same <tt>PartialSigAgg</tt> step as the final signer if it has not
 already been done.
 
-Otherwise, the resulting signature is a BIP 340 compatible signature and finalizers should treat it
-as such.
+Otherwise, the resulting signature is a [[bip-0340.mediawiki|BIP 340]] compatible signature and
+finalizers should treat it as such.
```

Comment:
```suggestion
Otherwise, the resulting signature is a [[bip-0340.mediawiki|BIP 340]] compatible signature and finalizers should treat it
as such.
```